src/validation/custom.validation.ts

```diff
@@ -279,6 +280,31 @@ export const checkBeneficialOwnerType = (beneficialOwnersStatement: string, valu
   return true;
 };
 
+export const checkBeneficialOwnerSubmission = (req) => {
+  const appData: ApplicationData = getApplicationData(req.session);
+  if (appData.beneficial_owners_statement === BeneficialOwnersStatementType.SOME_IDENTIFIED_ALL_DETAILS) {
+    if (!hasBeneficialOwners(appData)) {
```

Comment:
Does this work for AC5 in the story? It says we should show the error even if only one Corporate BO is added.

Reply:
AC 5 should read corporate MO not BO as AC2 covers this.

Reply:
AC updated after discussion with Karen. Code now matches requirements.